ScienceMotorController/ScienceMotorController.ino

```diff
@@ -0,0 +1,161 @@
+// Science Motor Controller Software
+// 2022 Prometheus, Mars Rover Design Team
+// Written by Faye Squires
+// #RoveSoHard
+
+#include "ScienceMotorController.h"
+
+void setup()
+{
+  RoveComm.begin(RC_SCIENCEACTUATIONBOARD_FOURTHOCTET, &TCPServer);
+
+  // attach motor controllers to pins
+  sensZ.attach(MOTOR_1_IN_A, MOTOR_1_IN_B, MOTOR_1_PWM);
+  gantX.attach(MOTOR_2_IN_A, MOTOR_2_IN_B, MOTOR_2_PWM);
+  gantZ.attach(MOTOR_3_IN_A, MOTOR_3_IN_B, MOTOR_3_PWM);
+
+  // start watchdog for mocos
+  watchdog.attach(eStop());
+  watchdog.start(WATCHDOG_TIMEOUT);
+
+  // start telemetry timer
+  telemTimer.attach(telemetry());
+  telemTimer.start(ROVECOMM_UPDATE_RATE);
```

Comment:
The attach() functions should take a function pointer as an argument, right? Not the return value of the function?